aws-redis.yml

```diff
@@ -172,6 +172,18 @@ provision:
     field_name: maintenance_end_min
     details: The preferred maintenance end minute
     enum: *minute_enum
+  - field_name: backup_retention_limit
```

Comment:
thoughts: we may want to add this constraint:

```yml
    constraints:
      maximum: 35
```

to replicate the validation performed in the TF provider
```
	"snapshot_retention_limit": {
		Type:         schema.TypeInt,
		Optional:     true,
		ValidateFunc: validation.IntAtMost(35),
	},
```

```
// IntAtMost returns a SchemaValidateFunc which tests if the provided value
// is of type int and is at most max (inclusive)
func IntAtMost(max int) schema.SchemaValidateFunc {
.....
```